app/src/main/java/com/ivy/wallet/RootActivity.kt

```diff
@@ -157,7 +165,10 @@ class RootActivity : AppCompatActivity(), RootScreen {
                 }
             }
 
-            dateTimePicker.Content()
+            dateTimePicker.Content(
```

Comment:
Just wrap the dateTimePicjer.Content with the theme here. No need to pass all these params.